2021-10-02 23:44:58 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Desktop/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Desktop/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:75)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Desktop/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Desktop/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Desktop/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2021-10-02 23:45:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "PRODUCTO" no encontrada
Table "PRODUCTO" not found; SQL statement:
SELECT id,nombre,precio FROM producto [42102-200]
2021-10-02 23:48:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "PRODUCTO" no encontrada
Table "PRODUCTO" not found; SQL statement:
SELECT id,nombre,precio FROM producto [42102-200]
2021-10-03 18:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Conversi√≥n de datos fallida, convirtiendo "Cazarecompensas"
Data conversion error converting "Cazarecompensas" [22018-200]
2021-10-03 18:49:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "descripcion" no encontrada
Column "descripcion" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getDouble(JdbcResultSet.java:752)
	at ejemplo.Aplicacion.main(Aplicacion.java:107)
2021-10-05 09:47:32 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:81)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2021-10-05 10:02:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "UPDATE NAVE SET DESCRIPCION = 'CUCU' WHERE IDNAVE 1[*]"
Syntax error in SQL statement "UPDATE NAVE SET DESCRIPCION = 'CUCU' WHERE IDNAVE 1[*]"; SQL statement:
UPDATE NAVE SET DESCRIPCION = 'CUCU' WHERE IDNAVE 1 [42000-200]
2021-10-05 10:04:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "descripcion" no encontrada
Column "descripcion" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:316)
	at ejemplo.Aplicacion.consultarPersonaje(Aplicacion.java:220)
	at ejemplo.Aplicacion.main(Aplicacion.java:117)
2021-10-05 10:05:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "descripcion" no encontrada
Column "descripcion" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:316)
	at ejemplo.Aplicacion.consultarPersonaje(Aplicacion.java:220)
	at ejemplo.Aplicacion.main(Aplicacion.java:117)
2021-10-05 10:06:26 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:81)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2021-10-05 10:07:04 database: wrong user or password; user: "SA"
org.h2.message.DbException: Nombre de usuario √≥ password incorrecto
Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:309)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:289)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:785)
	at org.h2.server.web.WebApp.login(WebApp.java:1008)
	at org.h2.server.web.WebApp.process(WebApp.java:223)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario √≥ password incorrecto
Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 21 more
2021-10-18 13:00:15 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2_alumnado/UT10_ProyectoBaseH2_alumnado/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2_alumnado/UT10_ProyectoBaseH2_alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:85)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2_alumnado/UT10_ProyectoBaseH2_alumnado/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2_alumnado/UT10_ProyectoBaseH2_alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2_alumnado/UT10_ProyectoBaseH2_alumnado/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2023-03-14 17:17:15 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:C:/Users/anaes/Documents/NetBeansProjects15/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/anaes/Documents/NetBeansProjects15/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:681)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:229)
	at ejemplo.Aplicacion.main(Aplicacion.java:85)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:C:/Users/anaes/Documents/NetBeansProjects15/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/anaes/Documents/NetBeansProjects15/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/anaes/Documents/NetBeansProjects15/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2023-05-03 19:37:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "

CREATE TABLE IF NOT EXISTS ENTRENAMIENTO(PRIMARY KEY CODIGO[*] INT,COD_USUARIO INT , COD_ACTIVIDAD INT, FECHA DATE)
CREATE TABLE IF NOT EXISTS USUARIO(PRIMARY KEY CODIGO INT, NOMBRE VARCHAR(50))
CREATE TABLE IF NOT EXISTS ACTIVIDAD(PRIMARY KEY CODOGO INT ,TIPO VARCHAR(50), DURACION INT, DISTANCIA FLOAT )
 "; expected "HASH, ("; SQL statement:


CREATE TABLE IF NOT EXISTS ENTRENAMIENTO(PRIMARY KEY codigo INT,cod_usuario INT , cod_actividad INT, fecha DATE)
CREATE TABLE IF NOT EXISTS USUARIO(PRIMARY KEY codigo INT, nombre VARCHAR(50))
CREATE TABLE IF NOT EXISTS ACTIVIDAD(PRIMARY KEY codogo INT ,tipo VARCHAR(50), duracion INT, distancia FLOAT )
  [42001-200]
2023-05-10 17:01:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL IDENTITY PRIMARY KEY  , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT, FECHA LOCALDATE NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL IDENTITY PRIMARY KEY  , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT, fecha LocalDate NOT NULL); [42001-200]
2023-05-10 17:14:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "


CREATE TABLE USUARIO( CODIGO INT NOT NULL IDENTITY PRIMARY KEY ([*]CODIGO), NOMBRE VARCHAR(50) NOT NULL);
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL IDENTITY PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL IDENTITY PRIMARY KEY (CODIGO),TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "HASH, AUTO_INCREMENT, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:



CREATE TABLE USUARIO( codigo INT NOT NULL IDENTITY PRIMARY KEY (codigo), nombre VARCHAR(50) NOT NULL);
CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL IDENTITY PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL IDENTITY PRIMARY KEY (codigo),tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:14:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "


CREATE TABLE USUARIO( CODIGO INT NOT NULL IDENTITY PRIMARY KEY ([*]CODIGO), NOMBRE VARCHAR(50) NOT NULL);
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL IDENTITY PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL IDENTITY PRIMARY KEY (CODIGO),TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "HASH, AUTO_INCREMENT, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:



CREATE TABLE USUARIO( codigo INT NOT NULL IDENTITY PRIMARY KEY (codigo), nombre VARCHAR(50) NOT NULL);
CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL IDENTITY PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL IDENTITY PRIMARY KEY (codigo),tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:14:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "


CREATE TABLE USUARIO( CODIGO INT NOT NULL IDENTITY PRIMARY KEY ([*]CODIGO), NOMBRE VARCHAR(50) NOT NULL);
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL IDENTITY PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL IDENTITY PRIMARY KEY (CODIGO),TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "HASH, AUTO_INCREMENT, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:



CREATE TABLE USUARIO( codigo INT NOT NULL IDENTITY PRIMARY KEY (codigo), nombre VARCHAR(50) NOT NULL);
CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL IDENTITY PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL IDENTITY PRIMARY KEY (codigo),tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:15:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "


CREATE TABLE USUARIO( CODIGO INT NOT NULL IDENTITY PRIMARY KEY ([*]CODIGO), NOMBRE VARCHAR(50) NOT NULL);
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL IDENTITY PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL IDENTITY PRIMARY KEY (CODIGO),TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "HASH, AUTO_INCREMENT, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:



CREATE TABLE USUARIO( codigo INT NOT NULL IDENTITY PRIMARY KEY (codigo), nombre VARCHAR(50) NOT NULL);
CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL IDENTITY PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL IDENTITY PRIMARY KEY (codigo),tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:20:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "


CREATE TABLE USUARIO( CODIGO INT NOT NULL  PRIMARY KEY ([*]CODIGO), NOMBRE VARCHAR(50) NOT NULL);
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL  PRIMARY KEY (CODIGO),TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "HASH, AUTO_INCREMENT, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:



CREATE TABLE USUARIO( codigo INT NOT NULL  PRIMARY KEY (codigo), nombre VARCHAR(50) NOT NULL);
CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL  PRIMARY KEY (codigo),tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:22:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:24:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:25:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:26:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:26:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Juan" not found; SQL statement:
INSERT INTO USUARIO VALUES (1,"Juan"), (2,"Maria");
INSERT INTO ENTRENAMIENTO VALUES (1,1,1,"2023-05-08"), (2,1,2,"2023-04-10"),(3,2,3,"2023-03-01");
INSERT INTO ACTIVIDAD VALUES (1,"Carrera",30,3.5), (2,"Nataci√≥n",45,1.2),(3,"Ciclismo",60,12.3), (4,"Fuerza",50,0.0);

 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:26:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "ACTIVIDAD" not found; SQL statement:
SELECT * FROM ACTIVIDAD [42102-200]
2023-05-10 17:27:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Juan" not found; SQL statement:
INSERT INTO USUARIO VALUES (1,"Juan"), (2,"Maria");
INSERT INTO ENTRENAMIENTO VALUES (1,1,1,"2023-05-08"), (2,1,2,"2023-04-10"),(3,2,3,"2023-03-01");
INSERT INTO ACTIVIDAD VALUES (1,"Carrera",30,3.5), (2,"Nataci√≥n",45,1.2),(3,"Ciclismo",60,12.3), (4,"Fuerza",50,0.0);

 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:27:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Juan" not found; SQL statement:
INSERT INTO USUARIO VALUES (1,"Juan"), (2,"Maria");
INSERT INTO ENTRENAMIENTO VALUES (1,1,1,"2023-05-08"), (2,1,2,"2023-04-10"),(3,2,3,"2023-03-01");
INSERT INTO ACTIVIDAD VALUES (1,"Carrera",30,3.5), (2,"Nataci√≥n",45,1.2),(3,"Ciclismo",60,12.3), (4,"Fuerza",50,0.0);

 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:30:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Juan" not found; SQL statement:
INSERT INTO USUARIO VALUES (1,"Juan"), (2,"Maria");
INSERT INTO ENTRENAMIENTO VALUES (1,1,1,"2023-05-08"), (2,1,2,"2023-04-10"),(3,2,3,"2023-03-01");
INSERT INTO ACTIVIDAD VALUES (1,"Carrera",30,3.5), (2,"Nataci√≥n",45,1.2),(3,"Ciclismo",60,12.3), (4,"Fuerza",50,0.0);

 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:30:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "ACTIVIDAD" not found; SQL statement:
SELECT * FROM ACTIVIDAD [42102-200]
2023-05-10 17:32:53 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:681)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:229)
	at ejemplo.Aplicacion.main(Aplicacion.java:87)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2023-05-10 17:33:19 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:681)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:229)
	at ejemplo.Aplicacion.main(Aplicacion.java:87)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2023-05-10 17:34:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:34:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:34:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:35:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:36:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:37:26 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:681)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:229)
	at ejemplo.Aplicacion.main(Aplicacion.java:87)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2023-05-10 17:37:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:39:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:40:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:40:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:42:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:43:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE NOT NULL);
CREATE TABLE ACTIVIDAD(CODIGO INT NOT NULL PRIMARY KEY,TIPO VARCHAR(50), DURACION INT NOT NULL, DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT NOT NULL  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate NOT NULL);
CREATE TABLE ACTIVIDAD(codigo INT NOT NULL PRIMARY KEY,tipo VARCHAR(50), duracion INT NOT NULL, distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:43:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Juan" not found; SQL statement:
INSERT INTO USUARIO (codigo, nombre)VALUES(1,"Juan");
INSERT INTO USUARIO (codigo, nombre)VALUES(2,"Maria");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (1,1,1,"2023-05-08");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (2,1,2,"2023-04-10");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (3,2,3,"2023-03-01");
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (1,"Carrera",30,3.5);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (2,"Nataci√≥n",45,1.2);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (3,"Ciclismo",60,12.3);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (4,"Fuerza",50,0.0); [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:43:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT  PRIMARY KEY REFERENCES USUARIO(CODIGO) , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE);
CREATE TABLE ACTIVIDAD(CODIGO INT  PRIMARY KEY,TIPO VARCHAR(50), DURACION INT , DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT  PRIMARY KEY REFERENCES USUARIO(codigo) , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate);
CREATE TABLE ACTIVIDAD(codigo INT  PRIMARY KEY,tipo VARCHAR(50), duracion INT , distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:44:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Juan" not found; SQL statement:
INSERT INTO USUARIO (codigo, nombre)VALUES(1,"Juan");
INSERT INTO USUARIO (codigo, nombre)VALUES(2,"Maria");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (1,1,1,"2023-05-08");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (2,1,2,"2023-04-10");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (3,2,3,"2023-03-01");
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (1,"Carrera",30,3.5);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (2,"Nataci√≥n",45,1.2);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (3,"Ciclismo",60,12.3);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (4,"Fuerza",50,0.0); [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:45:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT , FOREIGN KEY COD_USUARIO[*] INT ,FOREIGN KEY COD_ACTIVIDAD INT REFERENCES ACTIVIDAD(CODIGO), FECHA LOCALDATE);
CREATE TABLE ACTIVIDAD(CODIGO INT  PRIMARY KEY,TIPO VARCHAR(50), DURACION INT , DISTANCIA FLOAT NOT NULL);"; expected "("; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT , FOREIGN KEY cod_usuario INT ,FOREIGN KEY cod_actividad INT REFERENCES ACTIVIDAD(codigo), fecha LocalDate);
CREATE TABLE ACTIVIDAD(codigo INT  PRIMARY KEY,tipo VARCHAR(50), duracion INT , distancia FLOAT NOT NULL); [42001-200]
2023-05-10 17:46:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Unknown data type: "LOCALDATE"; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT , cod_usuario INT ,cod_actividad INT, fecha LocalDate);
CREATE TABLE ACTIVIDAD(codigo INT  PRIMARY KEY,tipo VARCHAR(50), duracion INT , distancia FLOAT NOT NULL); [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseCreateTable(Parser.java:8379)
	at org.h2.command.Parser.parseCreate(Parser.java:6276)
	at org.h2.command.Parser.parsePrepared(Parser.java:903)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:54)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.crearTablas(Aplicacion.java:190)
	at ejemplo.Aplicacion.main(Aplicacion.java:111)
2023-05-10 17:46:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Unknown data type: "LOCALDATE"; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT , cod_usuario INT ,cod_actividad INT, fecha LocalDate);
CREATE TABLE ACTIVIDAD(codigo INT ,tipo VARCHAR(50), duracion INT , distancia FLOAT NOT NULL); [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseCreateTable(Parser.java:8379)
	at org.h2.command.Parser.parseCreate(Parser.java:6276)
	at org.h2.command.Parser.parsePrepared(Parser.java:903)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:54)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.crearTablas(Aplicacion.java:190)
	at ejemplo.Aplicacion.main(Aplicacion.java:111)
2023-05-10 17:46:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Unknown data type: "LOCALDATE"; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT , cod_usuario INT ,cod_actividad INT, fecha LocalDate);
CREATE TABLE ACTIVIDAD(codigo INT ,tipo VARCHAR(50), duracion INT , distancia FLOAT); [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseCreateTable(Parser.java:8379)
	at org.h2.command.Parser.parseCreate(Parser.java:6276)
	at org.h2.command.Parser.parsePrepared(Parser.java:903)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:54)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.crearTablas(Aplicacion.java:190)
	at ejemplo.Aplicacion.main(Aplicacion.java:111)
2023-05-10 17:46:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Unknown data type: "LOCALDATE"; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT , cod_usuario INT ,cod_actividad INT, fecha LocalDate);
CREATE TABLE ACTIVIDAD(codigo INT ,tipo VARCHAR(50), duracion INT , distancia FLOAT); [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseCreateTable(Parser.java:8379)
	at org.h2.command.Parser.parseCreate(Parser.java:6276)
	at org.h2.command.Parser.parsePrepared(Parser.java:903)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:54)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.crearTablas(Aplicacion.java:190)
	at ejemplo.Aplicacion.main(Aplicacion.java:111)
2023-05-10 17:47:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Juan" not found; SQL statement:
INSERT INTO USUARIO (codigo, nombre)VALUES(1,"Juan");
INSERT INTO USUARIO (codigo, nombre)VALUES(2,"Maria");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (1,1,1,"2023-05-08");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (2,1,2,"2023-04-10");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (3,2,3,"2023-03-01");
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (1,"Carrera",30,3.5);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (2,"Nataci√≥n",45,1.2);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (3,"Ciclismo",60,12.3);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (4,"Fuerza",50,0.0); [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:49:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Juan" not found; SQL statement:
INSERT INTO USUARIO (codigo, nombre)VALUES(1,"Juan");
INSERT INTO USUARIO (codigo, nombre)VALUES(2,"Maria");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (1,1,1,"2023-05-08");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (2,1,2,"2023-04-10");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (3,2,3,"2023-03-01");
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (1,"Carrera",30,3.5);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (2,"Nataci√≥n",45,1.2);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (3,"Ciclismo",60,12.3);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (4,"Fuerza",50,0.0); [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:51:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Juan" not found; SQL statement:
INSERT INTO USUARIO (codigo, nombre)VALUES(1,"Juan");
INSERT INTO USUARIO (codigo, nombre)VALUES(2,"Maria");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (1,1,1,"2023-05-08");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (2,1,2,"2023-04-10");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (3,2,3,"2023-03-01");
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (1,"Carrera",30,3.5);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (2,"Nataci√≥n",45,1.2);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (3,"Ciclismo",60,12.3);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (4,"Fuerza",50,0.0); [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:51:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT ,
 COD_USUARIO INT ,
COD_ACTIVIDAD INT, FECHA DATE,
PRIMARY KEY (CODIGO),
FOREIGN KEY (COD_USUARIO) REFERENCES USUARIO(CODIGO),
FOREIGN KEY (COD_ACTIVIDAD) REFERENCES ACTIVIDAD(CODIGO),
)[*];
CREATE TABLE ACTIVIDAD(CODIGO INT ,
TIPO VARCHAR(50),
 DURACION INT ,
 DISTANCIA FLOAT,
PRIMARY KEY (CODIGO) );"; expected "identifier"; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT ,
 cod_usuario INT ,
cod_actividad INT, fecha Date,
PRIMARY KEY (codigo),
FOREIGN KEY (cod_usuario) REFERENCES USUARIO(codigo),
FOREIGN KEY (cod_actividad) REFERENCES ACTIVIDAD(codigo),
);
CREATE TABLE ACTIVIDAD(codigo INT ,
tipo VARCHAR(50),
 duracion INT ,
 distancia FLOAT,
PRIMARY KEY (codigo) ); [42001-200]
2023-05-10 17:52:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "ACTIVIDAD" not found; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT ,
 cod_usuario INT ,
cod_actividad INT, fecha Date,
PRIMARY KEY (codigo),
FOREIGN KEY (cod_usuario) REFERENCES USUARIO(codigo),
FOREIGN KEY (cod_actividad) REFERENCES ACTIVIDAD(codigo)
); [42102-200]
2023-05-10 17:52:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT ,
COD_USUARIO INT ,
COD_ACTIVIDAD INT, FECHA DATE,
PRIMARY KEY (CODIGO),
FOREIGN KEY (COD_USUARIO) REFERENCES USUARIO(CODIGO),
                                                          
)[*];
CREATE TABLE ACTIVIDAD(CODIGO INT ,
TIPO VARCHAR(50),
 DURACION INT ,
 DISTANCIA FLOAT,
PRIMARY KEY (CODIGO) );"; expected "identifier"; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT ,
cod_usuario INT ,
cod_actividad INT, fecha Date,
PRIMARY KEY (codigo),
FOREIGN KEY (cod_usuario) REFERENCES USUARIO(codigo),
--FOREIGN KEY (cod_actividad) REFERENCES ACTIVIDAD(codigo)
);
CREATE TABLE ACTIVIDAD(codigo INT ,
tipo VARCHAR(50),
 duracion INT ,
 distancia FLOAT,
PRIMARY KEY (codigo) ); [42001-200]
2023-05-10 17:53:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Juan" not found; SQL statement:
INSERT INTO USUARIO (codigo, nombre)VALUES(1,"Juan");
INSERT INTO USUARIO (codigo, nombre)VALUES(2,"Maria");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (1,1,1,"2023-05-08");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (2,1,2,"2023-04-10");
INSERT INTO ENTRENAMIENTO(codigo,cod_usuario,cod_actividad) VALUES (3,2,3,"2023-03-01");
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (1,"Carrera",30,3.5);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (2,"Nataci√≥n",45,1.2);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (3,"Ciclismo",60,12.3);
INSERT INTO ACTIVIDAD (codigo, duracion, distancia) VALUES (4,"Fuerza",50,0.0); [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:53:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Juan" not found; SQL statement:
INSERT INTO USUARIO (codigo, nombre)VALUES(1,"Juan");
INSERT INTO USUARIO (codigo, nombre)VALUES(2,"Maria"); [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:54:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "JUAN" not found; SQL statement:
INSERT INTO USUARIO (codigo, nombre)VALUES(1,Juan);
INSERT INTO USUARIO (codigo, nombre)VALUES(2,Maria); [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:55:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "JUAN" not found; SQL statement:
INSERT INTO USUARIO VALUES(1,Juan);
 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:56:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Juan" not found; SQL statement:
INSERT INTO USUARIO VALUES(1,"Juan");
 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 17:56:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "COD_USUARIO" not found; SQL statement:
SELECT codigo,cod_usuario,cod_actividad, fecha FROM USUARIO [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Select.prepare(Select.java:1206)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at ejemplo.Aplicacion.consultarUsuarios(Aplicacion.java:295)
	at ejemplo.Aplicacion.main(Aplicacion.java:146)
2023-05-10 17:57:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column count does not match; SQL statement:
INSERT INTO USUARIO VALUES(1,'Juan'); [21002-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.dml.Insert.prepare(Insert.java:319)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:46)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:219)
	at ejemplo.Aplicacion.main(Aplicacion.java:114)
2023-05-10 18:04:26 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:681)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:229)
	at ejemplo.Aplicacion.main(Aplicacion.java:87)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2023-05-10 18:05:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "COD_USUARIO" not found; SQL statement:
SELECT codigo,cod_usuario,cod_actividad, fecha FROM USUARIO [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Select.prepare(Select.java:1206)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at ejemplo.Aplicacion.consultarUsuarios(Aplicacion.java:295)
	at ejemplo.Aplicacion.main(Aplicacion.java:146)
2023-05-11 08:21:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "COD_USUARIO" not found; SQL statement:
SELECT codigo,cod_usuario,cod_actividad, fecha FROM USUARIO [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Select.prepare(Select.java:1206)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at ejemplo.Aplicacion.consultarUsuarios(Aplicacion.java:298)
	at ejemplo.Aplicacion.main(Aplicacion.java:146)
2023-05-11 08:21:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "cod_usuario" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.consultarUsuarios(Aplicacion.java:302)
	at ejemplo.Aplicacion.main(Aplicacion.java:146)
2023-05-11 08:51:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "cod_usuario" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.consultarUsuarios(Aplicacion.java:305)
	at ejemplo.Aplicacion.main(Aplicacion.java:146)
2023-05-11 08:57:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "cod_usuario" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.consultarUsuarios(Aplicacion.java:305)
	at ejemplo.Aplicacion.main(Aplicacion.java:146)
2023-05-11 08:58:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "cod_usuario" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.consultarUsuarios(Aplicacion.java:305)
	at ejemplo.Aplicacion.main(Aplicacion.java:146)
2023-05-11 13:57:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "PRECIO" not found; SQL statement:
UPDATE ACTIVIDAD SET precio=? WHERE id=?; [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.readTableColumn(Parser.java:1274)
	at org.h2.command.Parser.parseUpdateSetClause(Parser.java:1325)
	at org.h2.command.Parser.parseUpdate(Parser.java:1290)
	at org.h2.command.Parser.parsePrepared(Parser.java:1005)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.modificarActividad(Aplicacion.java:301)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2023-05-11 13:59:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "ACTIVIDADAMODIFICAR" not found; SQL statement:
UPDATE ACTIVIDAD SET actividadAmodificar WHERE actividadAmodificar=actividadAmodificar [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.readTableColumn(Parser.java:1274)
	at org.h2.command.Parser.parseUpdateSetClause(Parser.java:1325)
	at org.h2.command.Parser.parseUpdate(Parser.java:1290)
	at org.h2.command.Parser.parsePrepared(Parser.java:1005)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.modificarActividad(Aplicacion.java:301)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2023-05-11 14:03:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "UPDATE ACTIVIDAD SET DURACION WHERE[*] TIPO=ACTIVIDADAMODIFICAR"; expected "., ="; SQL statement:
UPDATE ACTIVIDAD SET duracion WHERE tipo=actividadAmodificar [42001-200]
2023-05-11 14:04:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "UPDATE ACTIVIDAD SET DURACION WHERE[*] CODIGO=ACTIVIDADAMODIFICAR"; expected "., ="; SQL statement:
UPDATE ACTIVIDAD SET duracion WHERE codigo=actividadAmodificar [42001-200]
2023-05-11 14:06:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "ACTIVIDADAMODIFICAR" not found; SQL statement:
UPDATE ACTIVIDAD SET duracion= tecladoDos WHERE codigo=actividadAmodificar [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:191)
	at org.h2.command.dml.Update.prepare(Update.java:266)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.modificarActividad(Aplicacion.java:301)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2023-05-11 14:09:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "actividadAmodificar"; SQL statement:
UPDATE ACTIVIDAD SET duracion= 'tecladoDos' WHERE codigo='actividadAmodificar' [22018-200]
2023-05-11 18:50:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "ACTIVIDADAMODIFICAR" not found; SQL statement:
UPDATE ACTIVIDAD SET duracion= tecladoDos WHERE codigo=actividadAmodificar [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:191)
	at org.h2.command.dml.Update.prepare(Update.java:266)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.modificarActividad(Aplicacion.java:301)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2023-05-11 18:51:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "ACTIVIDADAMODIFICAR" not found; SQL statement:
UPDATE ACTIVIDAD SET duracion= tecladoDos WHERE codigo=actividadAmodificar [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:191)
	at org.h2.command.dml.Update.prepare(Update.java:266)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.modificarActividad(Aplicacion.java:302)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2023-05-11 18:51:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "ACTIVIDADAMODIFICAR" not found; SQL statement:
UPDATE ACTIVIDAD SET duracion= tecladoDos WHERE codigo=actividadAmodificar [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:191)
	at org.h2.command.dml.Update.prepare(Update.java:266)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.modificarActividad(Aplicacion.java:302)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2023-05-11 18:52:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "actividadAmodificar"; SQL statement:
update ACTIVIDAD set duracion= 'tecladoDos' WHERE codigo='actividadAmodificar' [22018-200]
2023-05-11 18:54:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "actividadAmodificar"; SQL statement:
update ACTIVIDAD set duracion= 'tecladoDos' WHERE codigo='actividadAmodificar' [22018-200]
2023-05-11 18:56:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Parameter "#2" is not set; SQL statement:
update ACTIVIDAD set duracion= ? WHERE codigo=? [90012-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.Parameter.checkSet(Parameter.java:83)
	at org.h2.command.Prepared.checkParameters(Prepared.java:182)
	at org.h2.command.CommandContainer.update(CommandContainer.java:187)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at ejemplo.Aplicacion.modificarActividad(Aplicacion.java:302)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2023-05-11 19:05:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "ID" not found; SQL statement:
DELETE FROM USUARIO WHERE id=? [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Delete.prepare(Delete.java:184)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.borrarUsuario(Aplicacion.java:354)
	at ejemplo.Aplicacion.main(Aplicacion.java:150)
2023-05-11 19:08:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "ID" not found; SQL statement:
DELETE FROM USUARIO WHERE id=? [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Delete.prepare(Delete.java:184)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.borrarUsuario(Aplicacion.java:354)
	at ejemplo.Aplicacion.main(Aplicacion.java:150)
2023-05-11 19:09:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Parameter "#2" is not set; SQL statement:
update ACTIVIDAD set duracion= ? WHERE codigo=? [90012-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.Parameter.checkSet(Parameter.java:83)
	at org.h2.command.Prepared.checkParameters(Prepared.java:182)
	at org.h2.command.CommandContainer.update(CommandContainer.java:187)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at ejemplo.Aplicacion.modificarActividad(Aplicacion.java:302)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2023-05-11 19:11:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "USUARIOBORRAR" not found; SQL statement:
DELETE FROM USUARIO WHERE codigo=usuarioBorrar [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:191)
	at org.h2.command.dml.Delete.prepare(Delete.java:184)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.borrarUsuario(Aplicacion.java:355)
	at ejemplo.Aplicacion.main(Aplicacion.java:150)
2023-05-11 19:12:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "USUARIOBORRAR" not found; SQL statement:
DELETE FROM USUARIO WHERE codigo=usuarioBorrar [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:191)
	at org.h2.command.dml.Delete.prepare(Delete.java:184)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.borrarUsuario(Aplicacion.java:355)
	at ejemplo.Aplicacion.main(Aplicacion.java:150)
2023-05-11 19:16:14 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:681)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:229)
	at ejemplo.Aplicacion.main(Aplicacion.java:88)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2023-05-11 19:16:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "1" for parameter "parameterIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1503)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:394)
	at ejemplo.Aplicacion.borrarUsuario(Aplicacion.java:356)
	at ejemplo.Aplicacion.main(Aplicacion.java:150)
2023-05-11 19:18:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "1" for parameter "parameterIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1503)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:394)
	at ejemplo.Aplicacion.borrarUsuario(Aplicacion.java:357)
	at ejemplo.Aplicacion.main(Aplicacion.java:150)
2023-05-11 19:21:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE FROM usuario WHERE codigo=1[*]'"; SQL statement:
DELETE FROM usuario WHERE codigo=1' [42000-200]
2023-05-12 07:47:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE *[*] FROM USUARIO WHERE CODIGO='1'"; expected "identifier"; SQL statement:
DELETE * FROM USUARIO WHERE codigo='1' [42001-200]
2023-05-12 10:26:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE  FROM USUARIO WHERE codigo=[*]'1"; SQL statement:
DELETE  FROM USUARIO WHERE codigo='1 [42000-200]
2023-05-12 10:26:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE FROM USUARIO WHERE codigo=[*]'1"; SQL statement:
DELETE FROM USUARIO WHERE codigo='1 [42000-200]
2023-05-12 12:43:06 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE *[*] FROM USUARIO WHERE CODIGO=1"; expected "identifier"; SQL statement:
DELETE * from USUARIO where CODIGO=1 [42001-200]
2023-05-12 12:43:13 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROM USUARIO 
DELETE *[*] FROM USUARIO WHERE CODIGO='1'"; SQL statement:
SELECT * FROM USUARIO 
DELETE * from USUARIO where CODIGO='1' [42000-200]
2023-05-12 12:43:24 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE *[*] FROM USUARIO WHERE CODIGO='1'"; expected "identifier"; SQL statement:
DELETE * from USUARIO where CODIGO='1' [42001-200]
2023-05-12 12:47:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Parameter "#2" is not set; SQL statement:
update ACTIVIDAD set duracion= ? WHERE codigo=? [90012-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.Parameter.checkSet(Parameter.java:83)
	at org.h2.command.Prepared.checkParameters(Prepared.java:182)
	at org.h2.command.CommandContainer.update(CommandContainer.java:187)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at ejemplo.Aplicacion.modificarActividad(Aplicacion.java:302)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2023-05-12 12:51:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Parameter "#2" is not set; SQL statement:
UPDATE ACTIVIDAD set duracion= ? WHERE codigo=? [90012-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.Parameter.checkSet(Parameter.java:83)
	at org.h2.command.Prepared.checkParameters(Prepared.java:182)
	at org.h2.command.CommandContainer.update(CommandContainer.java:187)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at ejemplo.Aplicacion.modificarActividad(Aplicacion.java:302)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2023-05-12 12:59:28 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE FROM USUARIO
ALTER TABLE[*] USUARIO DROP FOREIGN KEY CODIGO"; SQL statement:
DELETE from USUARIO
alter table USUARIO drop foreign key codigo [42000-200]
2023-05-12 12:59:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE FROM USUARIO WHERE CODIGO=1
ALTER[*] TABLE USUARIO DROP FOREIGN KEY CODIGO"; SQL statement:
DELETE from USUARIO where codigo=1
alter table USUARIO drop foreign key codigo [42000-200]
2023-05-12 13:00:47 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE FROM USUARIO WHERE CODIGO=1
ALTER[*] TABLE USUARIO DROP PRIMARY KEY CODIGO"; SQL statement:
DELETE from USUARIO where codigo=1
alter table USUARIO drop primary key codigo [42000-200]
2023-05-12 13:02:52 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROMUSUARIO[*]"; SQL statement:
select * fromUSUARIO [42000-200]
2023-05-12 13:03:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE FROM USUARIO WHERE NOMBRE='MARIA'
SELECT[*] * FROM USUARIO"; SQL statement:
DELETE from USUARIO where NOMBRE='MARIA'
select * from USUARIO [42000-200]
2023-05-12 13:03:36 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE *[*] FROM USUARIO WHERE NOMBRE='MARIA'"; expected "identifier"; SQL statement:
DELETE * from USUARIO where NOMBRE='MARIA' [42001-200]
2023-05-12 13:03:53 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "MARIA" not found; SQL statement:
DELETE  from USUARIO where NOMBRE=MARIA [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:191)
	at org.h2.command.dml.Delete.prepare(Delete.java:184)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-05-12 13:04:25 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE  FROM USUARIO WHERE NOMBRE='MARIA'
SELECT[*] * FROM USUARIO"; SQL statement:
DELETE  from USUARIO where NOMBRE='MARIA'
select * from USUARIO [42000-200]
2023-05-12 13:04:28 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE  FROM USUARIO WHERE NOMBRE='MARIA'
SELECT[*] * FROM USUARIO"; SQL statement:
DELETE  from USUARIO where NOMBRE='MARIA'
select * from USUARIO [42000-200]
2023-05-12 13:04:33 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE  FROM USUARIO WHERE NOMBRE='MARIA'
SELECT[*] * FROM USUARIO"; SQL statement:
DELETE  from USUARIO where NOMBRE='MARIA'
select * from USUARIO [42000-200]
2023-05-12 13:04:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE  FROM USUARIO WHERE NOMBRE='MARIA'
SELECT[*] * FROM USUARIO"; SQL statement:
DELETE  from USUARIO where NOMBRE='MARIA'
SELECT * from USUARIO [42000-200]
2023-05-12 13:04:46 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE  FROM USUARIO WHERE NOMBRE='MARIA'
SELECT[*] * FROM USUARIO"; SQL statement:
DELETE  from USUARIO where NOMBRE='MARIA'
SELECT * from USUARIO [42000-200]
2023-05-12 13:07:22 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "DISTANCIA" not found; SQL statement:
DELETE  FROM USUARIO WHERE DISTANCIA='3.5' [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Delete.prepare(Delete.java:184)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-05-12 13:07:35 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "DISTANCIA" not found; SQL statement:
DELETE  FROM USUARIO WHERE DISTANCIA='3,5' [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Delete.prepare(Delete.java:184)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-05-12 13:12:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "

CREATE TABLE USUARIO(
CODIGO INT ,
NOMBRE VARCHAR(50),
                        
)[*];

CREATE TABLE ACTIVIDAD(CODIGO INT ,
TIPO VARCHAR(50),
DURACION INT ,
DISTANCIA FLOAT,
                         
CREATE TABLE ENTRENAMIENTO(CODIGO INT ,
COD_USUARIO INT ,
COD_ACTIVIDAD INT, 
FECHA DATE,
                       
                                                       
                                                          
);
"; expected "identifier"; SQL statement:


CREATE TABLE USUARIO(
codigo INT ,
nombre VARCHAR(50),
-- PRIMARY KEY (codigo) 
);

CREATE TABLE ACTIVIDAD(codigo INT ,
tipo VARCHAR(50),
duracion INT ,
distancia FLOAT,
--PRIMARY KEY (codigo) );
CREATE TABLE ENTRENAMIENTO(codigo INT ,
cod_usuario INT ,
cod_actividad INT, 
fecha Date,
--PRIMARY KEY (codigo),
--FOREIGN KEY (cod_usuario) REFERENCES USUARIO(codigo),
--FOREIGN KEY (cod_actividad) REFERENCES ACTIVIDAD(codigo)
);
 [42001-200]
2023-05-12 13:16:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] ON DELETE CASCADE FROM USUARIO WHERE CODIGO=1"; SQL statement:
ON ON DELETE CASCADE FROM USUARIO WHERE CODIGO=1 [42000-200]
2023-05-12 13:17:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE CASCADE FROM USUARIO WHERE CODIGO=1"; SQL statement:
ON DELETE CASCADE FROM USUARIO WHERE CODIGO=1 [42000-200]
2023-05-12 13:20:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO=1"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO=1 [42000-200]
2023-05-12 13:24:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "

CREATE TABLE USUARIO(
CODIGO INT ,
NOMBRE VARCHAR(50),
PRIMARY KEY (CODIGO) ,
ON[*] DELETE CASCADE
);

CREATE TABLE ACTIVIDAD(
CODIGO INT ,
TIPO VARCHAR(50),
DURACION INT ,
DISTANCIA FLOAT,
PRIMARY KEY (CODIGO) 
);
CREATE TABLE ENTRENAMIENTO(CODIGO INT ,
COD_USUARIO INT ,
COD_ACTIVIDAD INT, 
FECHA DATE,
PRIMARY KEY (CODIGO),
FOREIGN KEY (COD_USUARIO) REFERENCES USUARIO(CODIGO),
FOREIGN KEY (COD_ACTIVIDAD) REFERENCES ACTIVIDAD(CODIGO)
);
"; expected "identifier"; SQL statement:


CREATE TABLE USUARIO(
codigo INT ,
nombre VARCHAR(50),
PRIMARY KEY (codigo) ,
ON DELETE CASCADE
);

CREATE TABLE ACTIVIDAD(
codigo INT ,
tipo VARCHAR(50),
duracion INT ,
distancia FLOAT,
PRIMARY KEY (codigo) 
);
CREATE TABLE ENTRENAMIENTO(codigo INT ,
cod_usuario INT ,
cod_actividad INT, 
fecha Date,
PRIMARY KEY (codigo),
FOREIGN KEY (cod_usuario) REFERENCES USUARIO(codigo),
FOREIGN KEY (cod_actividad) REFERENCES ACTIVIDAD(codigo)
);
 [42001-200]
2023-05-12 13:24:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "

CREATE TABLE USUARIO(
CODIGO INT ,
NOMBRE VARCHAR(50),
PRIMARY KEY (CODIGO) ,
ON[*] DELETE CASCADE
);

CREATE TABLE ACTIVIDAD(
CODIGO INT ,
TIPO VARCHAR(50),
DURACION INT ,
DISTANCIA FLOAT,
PRIMARY KEY (CODIGO) 
);
CREATE TABLE ENTRENAMIENTO(CODIGO INT ,
COD_USUARIO INT ,
COD_ACTIVIDAD INT, 
FECHA DATE,
PRIMARY KEY (CODIGO),
FOREIGN KEY (COD_USUARIO) REFERENCES USUARIO(CODIGO),
FOREIGN KEY (COD_ACTIVIDAD) REFERENCES ACTIVIDAD(CODIGO)
);
"; expected "identifier"; SQL statement:


CREATE TABLE USUARIO(
codigo INT ,
nombre VARCHAR(50),
PRIMARY KEY (codigo) ,
ON DELETE CASCADE
);

CREATE TABLE ACTIVIDAD(
codigo INT ,
tipo VARCHAR(50),
duracion INT ,
distancia FLOAT,
PRIMARY KEY (codigo) 
);
CREATE TABLE ENTRENAMIENTO(codigo INT ,
cod_usuario INT ,
cod_actividad INT, 
fecha Date,
PRIMARY KEY (codigo),
FOREIGN KEY (cod_usuario) REFERENCES USUARIO(codigo),
FOREIGN KEY (cod_actividad) REFERENCES ACTIVIDAD(codigo)
);
 [42001-200]
2023-05-12 13:24:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "

CREATE TABLE USUARIO(
CODIGO INT ,
NOMBRE VARCHAR(50),
PRIMARY KEY (CODIGO) ,
ON[*] DELETE CASCADE
);

CREATE TABLE ACTIVIDAD(
CODIGO INT ,
TIPO VARCHAR(50),
DURACION INT ,
DISTANCIA FLOAT,
PRIMARY KEY (CODIGO) 
);
CREATE TABLE ENTRENAMIENTO(CODIGO INT ,
COD_USUARIO INT ,
COD_ACTIVIDAD INT, 
FECHA DATE,
PRIMARY KEY (CODIGO),
FOREIGN KEY (COD_USUARIO) REFERENCES USUARIO(CODIGO),
FOREIGN KEY (COD_ACTIVIDAD) REFERENCES ACTIVIDAD(CODIGO)
);
"; expected "identifier"; SQL statement:


CREATE TABLE USUARIO(
codigo INT ,
nombre VARCHAR(50),
PRIMARY KEY (codigo) ,
ON DELETE CASCADE
);

CREATE TABLE ACTIVIDAD(
codigo INT ,
tipo VARCHAR(50),
duracion INT ,
distancia FLOAT,
PRIMARY KEY (codigo) 
);
CREATE TABLE ENTRENAMIENTO(codigo INT ,
cod_usuario INT ,
cod_actividad INT, 
fecha Date,
PRIMARY KEY (codigo),
FOREIGN KEY (cod_usuario) REFERENCES USUARIO(codigo),
FOREIGN KEY (cod_actividad) REFERENCES ACTIVIDAD(codigo)
);
 [42001-200]
2023-05-12 13:24:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "

CREATE TABLE USUARIO(
CODIGO INT ,
NOMBRE VARCHAR(50),
PRIMARY KEY (CODIGO) 
ON[*] DELETE CASCADE
);

CREATE TABLE ACTIVIDAD(
CODIGO INT ,
TIPO VARCHAR(50),
DURACION INT ,
DISTANCIA FLOAT,
PRIMARY KEY (CODIGO) 
);
CREATE TABLE ENTRENAMIENTO(CODIGO INT ,
COD_USUARIO INT ,
COD_ACTIVIDAD INT, 
FECHA DATE,
PRIMARY KEY (CODIGO),
FOREIGN KEY (COD_USUARIO) REFERENCES USUARIO(CODIGO),
FOREIGN KEY (COD_ACTIVIDAD) REFERENCES ACTIVIDAD(CODIGO)
);
"; expected "INDEX, ,, )"; SQL statement:


CREATE TABLE USUARIO(
codigo INT ,
nombre VARCHAR(50),
PRIMARY KEY (codigo) 
ON DELETE CASCADE
);

CREATE TABLE ACTIVIDAD(
codigo INT ,
tipo VARCHAR(50),
duracion INT ,
distancia FLOAT,
PRIMARY KEY (codigo) 
);
CREATE TABLE ENTRENAMIENTO(codigo INT ,
cod_usuario INT ,
cod_actividad INT, 
fecha Date,
PRIMARY KEY (codigo),
FOREIGN KEY (cod_usuario) REFERENCES USUARIO(codigo),
FOREIGN KEY (cod_actividad) REFERENCES ACTIVIDAD(codigo)
);
 [42001-200]
2023-05-12 13:25:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO=1"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO=1 [42000-200]
2023-05-12 13:25:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "

CREATE TABLE ACTIVIDAD(
CODIGO INT ,
TIPO VARCHAR(50),
DURACION INT ,
DISTANCIA FLOAT,
PRIMARY KEY (CODIGO) 
ON[*] DELETE CASCADE
);
CREATE TABLE ENTRENAMIENTO(CODIGO INT ,
COD_USUARIO INT ,
COD_ACTIVIDAD INT, 
FECHA DATE,
PRIMARY KEY (CODIGO),
FOREIGN KEY (COD_USUARIO) REFERENCES USUARIO(CODIGO),
FOREIGN KEY (COD_ACTIVIDAD) REFERENCES ACTIVIDAD(CODIGO)
ON DELETE CASCADE
);
"; expected "INDEX, ,, )"; SQL statement:


CREATE TABLE ACTIVIDAD(
codigo INT ,
tipo VARCHAR(50),
duracion INT ,
distancia FLOAT,
PRIMARY KEY (codigo) 
ON DELETE CASCADE
);
CREATE TABLE ENTRENAMIENTO(codigo INT ,
cod_usuario INT ,
cod_actividad INT, 
fecha Date,
PRIMARY KEY (codigo),
FOREIGN KEY (cod_usuario) REFERENCES USUARIO(codigo),
FOREIGN KEY (cod_actividad) REFERENCES ACTIVIDAD(codigo)
ON DELETE CASCADE
);
 [42001-200]
2023-05-12 13:26:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT ,
COD_USUARIO INT ,
COD_ACTIVIDAD INT, 
FECHA DATE,
PRIMARY KEY (CODIGO)ON[*] DELETE CASCADE,
FOREIGN KEY (COD_USUARIO) REFERENCES USUARIO(CODIGO),
FOREIGN KEY (COD_ACTIVIDAD) REFERENCES ACTIVIDAD(CODIGO)

);
"; expected "INDEX, ,, )"; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT ,
cod_usuario INT ,
cod_actividad INT, 
fecha Date,
PRIMARY KEY (codigo)ON DELETE CASCADE,
FOREIGN KEY (cod_usuario) REFERENCES USUARIO(codigo),
FOREIGN KEY (cod_actividad) REFERENCES ACTIVIDAD(codigo)

);
 [42001-200]
2023-05-12 13:27:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO=1"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO=1 [42000-200]
2023-05-12 13:27:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO=1"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO=1 [42000-200]
2023-05-12 13:27:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT ,
COD_USUARIO INT ,
COD_ACTIVIDAD INT, 
FECHA DATE,
PRIMARY KEY (CODIGO)ON[*] DELETE CASCADE,
FOREIGN KEY (COD_USUARIO) REFERENCES USUARIO(CODIGO) ON DELETE CASCADE,
FOREIGN KEY (COD_ACTIVIDAD) REFERENCES ACTIVIDAD(CODIGO)

);
"; expected "INDEX, ,, )"; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT ,
cod_usuario INT ,
cod_actividad INT, 
fecha Date,
PRIMARY KEY (codigo)ON DELETE CASCADE,
FOREIGN KEY (cod_usuario) REFERENCES USUARIO(codigo) ON DELETE CASCADE,
FOREIGN KEY (cod_actividad) REFERENCES ACTIVIDAD(codigo)

);
 [42001-200]
2023-05-12 13:28:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO=1"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO=1 [42000-200]
2023-05-12 13:29:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO=1"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO=1 [42000-200]
2023-05-12 13:31:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO=1"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO=1 [42000-200]
2023-05-12 13:32:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO=1"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO=1 [42000-200]
2023-05-12 13:32:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE ENTRENAMIENTO(CODIGO INT ,
COD_USUARIO INT ,
COD_ACTIVIDAD INT, 
FECHA DATE,
PRIMARY KEY (CODIGO) ON[*] UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (COD_USUARIO) REFERENCES USUARIO(CODIGO) ON UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (COD_ACTIVIDAD) REFERENCES ACTIVIDAD(CODIGO) ON UPDATE CASCADE ON DELETE CASCADE

);
"; expected "INDEX, ,, )"; SQL statement:

CREATE TABLE ENTRENAMIENTO(codigo INT ,
cod_usuario INT ,
cod_actividad INT, 
fecha Date,
PRIMARY KEY (codigo) ON UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (cod_usuario) REFERENCES USUARIO(codigo) ON UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (cod_actividad) REFERENCES ACTIVIDAD(codigo) ON UPDATE CASCADE ON DELETE CASCADE

);
 [42001-200]
2023-05-12 13:34:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO=1"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO=1 [42000-200]
2023-05-12 13:37:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO=1"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO=1 [42000-200]
2023-05-12 13:37:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO='1'"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO='1' [42000-200]
2023-05-12 13:40:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO=1"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO=1 [42000-200]
2023-05-12 13:41:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO=USUARIOBORRAR"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO=usuarioBorrar [42000-200]
2023-05-12 13:42:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ON[*] DELETE FROM USUARIO WHERE CODIGO=2"; SQL statement:
ON DELETE FROM USUARIO WHERE CODIGO=2 [42000-200]
2023-05-12 13:43:54 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROMUSUARIO[*]"; SQL statement:
select * fromUSUARIO [42000-200]
2023-05-12 13:50:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Parameter "#2" is not set; SQL statement:
UPDATE ACTIVIDAD set duracion= ? WHERE codigo=? [90012-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.Parameter.checkSet(Parameter.java:83)
	at org.h2.command.Prepared.checkParameters(Prepared.java:182)
	at org.h2.command.CommandContainer.update(CommandContainer.java:187)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at ejemplo.Aplicacion.modificarActividad(Aplicacion.java:302)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2023-05-12 18:29:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "UPDATE ACTIVIDAD SET DURACION= ?,DISTANCIA?[*] WHERE CODIGO=?"; expected "., ="; SQL statement:
UPDATE ACTIVIDAD SET duracion= ?,distancia? WHERE codigo=? [42001-200]
2023-05-12 18:30:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "UPDATE ACTIVIDAD SET DURACION= ?,DISTANCIA?[*] WHERE CODIGO=?"; expected "., ="; SQL statement:
UPDATE ACTIVIDAD SET duracion= ?,distancia? WHERE codigo=? [42001-200]
2023-05-12 18:31:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "UPDATE ACTIVIDAD SET DURACION= ?,DISTANCIA?[*] WHERE CODIGO=?"; expected "., ="; SQL statement:
UPDATE ACTIVIDAD SET duracion= ?,distancia? WHERE codigo=? [42001-200]
2023-05-12 18:33:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "UPDATE ACTIVIDAD SET DURACION= ?,DISTANCIA?[*] WHERE CODIGO=?"; expected "., ="; SQL statement:
UPDATE ACTIVIDAD SET duracion= ?,distancia? WHERE codigo=? [42001-200]
2023-05-13 12:26:21 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:681)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:229)
	at ejemplo.Aplicacion.main(Aplicacion.java:87)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2023-05-13 12:43:59 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:681)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:229)
	at ejemplo.Aplicacion.main(Aplicacion.java:87)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/JCMCarra/OneDrive - Unit4/Documents/NetBeansProjects/PROG10_Tarea_2022_23_Alumnado/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
